wallet/address.js

```diff
@@ -183,33 +182,34 @@ function decryptDerivationPath(addressPayload, hdPassphrase) {
   }
 }
 
-function getCheckSum(input) {
-  return crc32.buf(input) >>> 0
-}
+const crc32Unsigned = (input) => crc32.buf(input) >>> 0
 
 async function deriveHdPassphrase(hdNode) {
-  const extendedPublicKey = new Buffer(hdNode.getPublicKey() + hdNode.getChainCode(), 'hex')
+  const extendedPublicKey = Buffer.concat([
```

Comment:
why not make `getExtendedPublicKey` a method of `hdnode` ?

Reply:
ok

Reply:
This is the only occurrence. I saw no profit,  but it runs many times, so ok